custom_endpoints/README.md

```diff
@@ -0,0 +1,32 @@
+# Overview of custom REST endpoints
```

Comment:
There is considerable overlap between this readme and [README.md](https://github.com/splunk/splunk-app-examples/tree/master/custom_endpoints/hello-world#readme).

Consolidate into this one? Or, remove the other one if we no longer need it?

Reply:
Also applies to the other folders existing readmes (custom_search_commands, modularinputs, setup_pages) I suppose

Reply:
Yea, that's a good point. We wanted a root-level readme for each extension point, but in this case, there's only the one example, so somewhat redundant. I still think there is enough custom REST endpoint-general information in this readme to justify leaving them both, especially if we create additional examples in the future.